test/lib/ansible_test/_data/requirements/ansible.txt

```diff
@@ -7,6 +7,9 @@ jinja2 >= 3.0.0
 PyYAML >= 5.1  # PyYAML 5.1 is required for Python 3.8+ support
 cryptography
 packaging
+# importlib.resources in stdlib for py3.9 is lacking native hooks for
+# importlib.resources.files
+importlib_resources >= 5.0, < 5.1; python_version < '3.10'
```

Comment:
As of now, nothing in core uses `importlib.resources`, but the intention is that we will.  We can rip this out before the 3.15 release, and just move it to `units.txt` if we end up not using this ourselves.

Reply:
Ok, now we're using it.